Update multi-project docs for v1.5.0-rc1 (!) (#3171)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-properties/latest-version.md

---
resource_types: [models]
datatype: latest_version
required: no
---

<File name='models/<schema>.yml'>

```yml
models:
  - name: model_name
    latest_version: 2
    [versions](resource-properties/versions):
      - v: 2
      - v: 1
```

</File>

## Definition

The latest version of this model. The "latest" version is relevant for:
1. Resolving `ref()` calls to this model that are "unpinned" (a version is not explicitly specified)
2. Selecting model versions using the [`version:` selection method](methods#the-version-method), based on whether a given model version is `latest`, `prerelease`, or `old`

> This value can be either a string or numeric (integer or float) value, and must be one of the [version identifiers](resource-properties/versions#v) specified in this model's list of `versions`.

Reviewer: ```suggestion
This value can be a string or a numeric (integer or float) value. It must be one of the [version identifiers](resource-properties/versions#v) specified in this model's list of `versions`.
```

## Default

If not specified for a versioned model, `latest_version` defaults to the largest [version identifier](resource-properties/versions#v): numerically greatest (if all version identifiers are numeric), otherwise the alphabetically last (if they are strings).

For a non-versioned model (no `versions` list), `latest_version` has no value.

> If `latest_version` is not specified for a versioned model, `latest_version` defaults to the largest 

Reviewer: ```suggestion
If `latest_version` is not specified for a versioned model, `latest_version` defaults to the largest.
```
